File: java/program/PetProduct.java

public class PetProduct implements Food, Toy {
    private String brand;
    private String type;
    private String material;
    private String targetAge;
    private float price;
    private int stockQuantity;
    
    public PetProduct() {
    }
    
    public PetProduct(String brand, String type, String material, String targetAge, float price, int stockQuantity) {
        this.brand = brand;
        this.type = type;
        this.material = material;
        this.targetAge = targetAge;
        this.price = price;
        this.stockQuantity = stockQuantity;
    }
    
    @Override
    public void setBrand(String brand) {
        this.brand = brand;
    }
    
    @Override
    public void setType(String type) {
        this.type = type;
    }
    
    @Override
    public String getBrand() {
        return this.brand;
    }
    
    @Override
    public String getType() {
        return this.type;
    }
    
    @Override
    public void setMaterial(String material) {
        this.material = material;
    }
    
    @Override
    public void setTargetAge(String targetAge) {
        this.targetAge = targetAge;
    }
    
    @Override
    public String getMaterial() {
        return this.material;
    }
    
    @Override
    public String getTargetAge() {
        return this.targetAge;
    }
    
    public void setPrice(float price) {
        this.price = price;
    }
    
    public void setStockQuantity(int stockQuantity) {
        this.stockQuantity = stockQuantity;
    }
    
    public float getPrice() {
        return this.price;
    }
    
    public int getStockQuantity() {
        return this.stockQuantity;
    }
}

File: python/program/PetProduct.py

from Food import Food
from Toy import Toy

class PetProduct(Food, Toy):
    def __init__(self, brand="", type="", material="", target_age="", price=0.0, stock_quantity=0):
        Food.__init__(self, brand, type)
        Toy.__init__(self, material, target_age)
        self.__price = price
        self.__stock_quantity = stock_quantity
    
    def set_price(self, price):
        self.__price = price
    
    def set_stock_quantity(self, stock_quantity):
        self.__stock_quantity = stock_quantity
    
    def get_price(self):
        return self.__price
    
    def get_stock_quantity(self):
        return self.__stock_quantity